src/translator/response.h

```diff
@@ -70,6 +70,9 @@ struct Response {
   /// to (sub-)words accessible through Annotation.
   std::vector<Alignment> alignments;
 
+  /// Tag positions at char-level in the translated text (in the same order as in the source text)
+  std::vector<ByteRange> tagPositionTarget;
```

Comment:
There is a `AnnotatedText::tagPosition` available. Please use it. If you think about it these HTML tags are just an additional annotation on target.